aria-datepicker/src/DatePicker.tsx

```diff
@@ -65,7 +81,13 @@ export function DatePicker<T extends DateValue = DateValue>(
           {...popoverClasses}
         >
           <Dialog {...dialogProps} {...dialogClasses}>
-            <Calendar {...calendarProps} {...calenderClasses} />
+            {hasTime && !props.noTimeField ? (
```

Comment:
Not sure what's going on here, why is there a time field inside the dialog? Time editing is done in the DateField..